src/main/java/ladder/domain/ladder/line/point/LinePoint.java

```diff
@@ -55,4 +38,8 @@ public void connect (LinePoint targetPoint) {
     this.pointDirection = LadderPointDirection.RIGHT;
     targetPoint.pointDirection = LadderPointDirection.LEFT;
   }
+
+  public LinePoint next() {
+    return new LinePoint(index+1);
```

Comment:
1도 매직넘버니 분리해주면 더 좋을 것 같네요
```suggestion
    return new LinePoint(index+NEXT_POISITION_INDEX);
```